src/Listeners/LoadSettingsFromDatabase.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace ReFlar\LevelRanks\Listeners;
+use Flarum\Api\Event\Serializing;
+use Flarum\Api\Serializer\ForumSerializer;
+use Flarum\Settings\SettingsRepositoryInterface;
+use Illuminate\Contracts\Events\Dispatcher;
+class LoadSettingsFromDatabase
+{
+    /**
+     * @var string
+     */
+    protected $packagePrefix = 'reflar-level-ranks.';
+    /**
+     * @var array
+     */
+    protected $fieldsToGet = [
+        'pointsText'
+    ];
+    /**
+     * @var SettingsRepositoryInterface
+     */
+    protected $settings;
+    /**
+     * LoadSettingsFromDatabase constructor.
+     *
+     * @param SettingsRepositoryInterface $settings
+     */
+    public function __construct(SettingsRepositoryInterface $settings)
+    {
+        $this->settings = $settings;
+    }
+    /**
+     * @param Dispatcher $events
+     */
+    public function subscribe(Dispatcher $events)
+    {
+        $events->listen(Serializing::class, [$this, 'prepareApiAttributes']);
+    }
+    /**
+     * @param Serializing $event
+     */
+    public function prepareApiAttributes(Serializing $event)
+    {
+        if ($event->isSerializer(ForumSerializer::class)) {
+            foreach ($this->fieldsToGet as $field) {
+                $value = $this->settings->get($this->packagePrefix.$field);
```

Comment:
What is all this?

Reply:
Also styleCi

Reply:
@CDK2020 what would you replace that with?

Reply:
Looks how it is done here: https://github.com/ReFlar/twofactor/blob/master/src/Listeners/AddApiAttributes.php

Your solution is great when adding tons of things (like in cookie consent) and I'm totally stealing this for the future, but for 1 key, it's bloated.

Reply:
@CDK2020 fixed now